ArchiSteamFarm/Json/Steam.cs

```diff
@@ -476,6 +480,9 @@ internal sealed class InventoryResponse : NumberResponse {
 			private InventoryResponse() { }
 
 			internal sealed class Description {
+				[JsonExtensionData]
+				internal Dictionary<string, JToken> AdditionalProperties;
```

Comment:
Should be `readonly` if it's not meant to be set. Also use `ImmutableDictionary` if it's not meant for us to add anything there.